### ISSUE TITLE
Does not support localization

### ISSUE THREAD

**Issue by tjnurmin** [okfn/ckanjs-deprecated#36]

**Comment by rufuspollock** [comment]
Good point. We'll try and look into this. Any recommendations for i18n approaches for js (we've done simple js i18n in ckan core and in annotator - http://github.com/okfn/annotator)
